ch_6_GPU_Acceleration/6_2_GPU_Baichuan2-7B.md

```diff
@@ -0,0 +1,117 @@
+# 6.2 Run Baichuan 2 (7B) on Intel GPUs
+
+You can use IPEX-LLM to load any ModelScope model for acceleration on Intel GPUs. With IPEX-LLM, PyTorch models (in FP16/BF16/FP32) hosted on ModelScope can be loaded and optimized automatically on Intel GPUs with low-bit quantization (supported precisions include INT4/NF4/INT5/FP8/INT8).
+
+In this tutorial, you will learn how to run LLMs on Intel GPUs with IPEX-LLM optimizations, and based on that build a stream chatbot. A popular open-source LLM [baichuan-inc/Baichuan2-7B-Chat](https://www.modelscope.cn/models/baichuan-inc/Baichuan2-7B-Chat) is used as an example.
+
+> [!NOTE]
+> Please make sure that you have prepared the environment for IPEX-LLM on GPU before you started. Refer to [here](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html) for more information regarding installation and environment preparation.
+
+
+## 6.2.1 Load Model in Low Precision
+
+One common use case is to load a model from ModelScope hub with IPEX-LLM low-bit precision optimization. For Baichuan 2 (7B), you could simply import `ipex_llm.transformers.AutoModelForCausalLM` instead of `transformers.AutoModelForCausalLM`, and specify `load_in_4bit=True` or `load_in_low_bit` parameter accordingly in the `from_pretrained` function.
+
+For Intel GPUs, **once you have the model in low precision, set it to `to('xpu')`.**
+
+**For INT4 Optimizations (with `load_in_4bit=True`):**
+
+```python
+from ipex_llm.transformers import AutoModelForCausalLM
+
+model_in_4bit = AutoModelForCausalLM.from_pretrained(pretrained_model_name_or_path="baichuan-inc/Baichuan2-7B-Chat",
```

Comment:
some model has different model id in huggingface and modelscope, should we add some comment here?